web/static/js/snake.js

```diff
@@ -2,10 +2,117 @@ import $ from "jquery";
 import Vue from "vue";
 import * as THREE from "three";
 import * as TWEEN from "tween.js";
+import { Howler, Howl } from 'howler';
 
 var SNAKE_HEAD_URL = "/images/snake_head_nerdy.png";
 var SNAKE_HEAD_SCALE = 1.5;
 var FOOD_URL = "/images/food.png"
+var AUDIO_URL = "/audio/"
+var AUDIO_PLAYLISTS_URL = AUDIO_URL + "playlists.json"
```

Comment:
This code is shared with another project which isn't ES6, so I'm maintaining Original JavaScript (TM) compatibility